### ISSUE TITLE
训练完成后嘴部出现噪点方块，请问大佬怎么解决

### ISSUE THREAD

**Issue by lsjallk** [anliyuan/Ultralight-Digital-Human#146]
我是搭建在comfy UI中运行的，现在情况是可以正常运行，推理可以正常进行，但是训练时会很快结束，并且没有错误提示，然后用模型进行推理就会出现一个彩色噪点的方块在嘴部，请问是什么原因？

<img width="492" alt="Image" src="https://github.com/user-attachments/assets/d7ad1eff-ea45-4518-8806-afbbf3018e0e" />

**Comment by anliyuan** [comment]
复制一下日志把

**Comment by lsjallk** [comment]
今天复现了之前发生的情况，日志中没有错误信息输出，设定的epoch是200 跑到4直接终止，然后模型中就出现了和之前相同的嘴部彩色方块，还请大佬指点
`detecting landmarks...: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6946/6946 [30:01<00:00,  3.86it/s]
extracting audio feature...
/home/fox/udh-fine/models/AIFSH/UltralightDigitalHuman/hubert-large-ls960-ft
Loading the Wav2Vec2 Processor...
Loading the HuBERT Model...
SR: 16000 to 16000
(6946, 2, 1024)
/home/fox/xudh/bin/python /home/fox/udh-fine/custom_nodes/UltralightDigitalHuman-ComfyUI/udh/syncnet.py --save_dir /home/fox/udh-fine/output/UltralightDigitalHuman/wl-4@hubert/syncnet_ckpt --dataset_dir /home/fox/udh-fine/output/UltralightDigitalHuman/wl-4@hubert/dataset                 --asr hubert --patient 3 --batch_size 16
training syncnet: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 40/40 [09:56<00:00, 14.92s/it, epoch=40, loss (batch)=0.000127]
min loss is 0.00012744232182907496 at epoch 39
/home/fox/xudh/bin/python /home/fox/udh-fine/custom_nodes/UltralightDigitalHuman-ComfyUI/udh/train.py --save_dir /home/fox/udh-fine/output/UltralightDigitalHuman/wl-4@hubert/checkpoint --dataset_dir /home/fox/udh-fine/output/UltralightDigitalHuman/wl-4@hubert/dataset                 --asr hubert --use_syncnet --syncnet_checkpoint /home/fox/udh-fine/output/UltralightDigitalHuman/wl-4@hubert/syncnet_ckpt/min_loss.pth                     --epochs 200 --batchsize 16 --lr 0.0 --patient 3
/home/fox/xudh/lib/python3.12/site-packages/torchvision/models/_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
/home/fox/xudh/lib/python3.12/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=VGG19_Weights.IMAGENET1K_V1`. You can also use `weights=VGG19_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
/home/fox/udh-fine/custom_nodes/UltralightDigitalHuman-ComfyUI/udh/train.py:74: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  syncnet.load_state_dict(torch.load(args.syncnet_checkpoint))
/home/fox/udh-fine/output/UltralightDigitalHuman/wl-4@hubert/dataset
(6946, 2, 1024)
6946
Epoch 1/200: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6945/6945 [00:50<00:00, 136.77img/s, loss (batch)=0.461]
Epoch 1 Loss:0.45102183407750623
Epoch 2/200: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6945/6945 [00:50<00:00, 137.67img/s, loss (batch)=0.438]
Epoch 2 Loss:0.45107027511487063
Epoch 3/200: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6945/6945 [00:50<00:00, 138.02img/s, loss (batch)=0.475]
Epoch 3 Loss:0.45148260894863085
Epoch 4/200: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6945/6945 [00:50<00:00, 138.33img/s, loss (batch)=0.464]
Epoch 4 Loss:0.45114990185047016
min loss is 0.45102183407750623 at epoch 1
/home/fox/udh-fine/custom_nodes/MaskGCT-ComfyUI/models/tts/maskgct/maskgct_utils.py:56: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  stat_mean_var = torch.load(os.path.join(os.path.dirname(__file__),"ckpt/wav2vec2bert_stats.pt"))
/home/fox/xudh/lib/python3.12/site-packages/torch/nn/utils/weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)

`

**Comment by anliyuan** [comment]
用一个100 epoch左右的ckpt试试呢？

**Comment by cacard** [comment]
这个“花图”其实就是训练时的刚启动时的 假数据 推理出来的，你这loss应该很高了，没收敛。
